sdk/apimanagement/arm-apimanagement/samples-dev/authorizationProviderCreateOrUpdateSample.ts

```diff
@@ -1,13 +1,6 @@
-/*
- * Copyright (c) Microsoft Corporation.
- * Licensed under the MIT License.
- *
- * Code generated by Microsoft (R) AutoRest Code Generator.
- * Changes may cause incorrect behavior and will be lost if the code is regenerated.
```

Comment:
@v-jiaodi Could you help double confirm if the old headers are replaced in HLC/RLC/Modular?

Reply:
HLC still use the old headers. https://github.com/Azure/autorest.typescript/blob/main/packages/autorest.typescript/src/typescriptGenerator.ts#L136